<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISedelsResult</class>
 <widget class="QMainWindow" name="UISedelsResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>658</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>658</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Результати розв'язку СЛАУ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblVarsAmount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Кількість змінних: 3</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>631</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="funcsList" stretch="0,0">
     <item>
      <widget class="QLabel" name="stub1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stub2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
